homeassistant/components/motionblinds_ble/strings.json

```diff
@@ -33,5 +33,18 @@
         "vertical": "Vertical blind"
       }
     }
+  },
+  "entity": {
+    "button": {
+      "connect": {
+        "name": "Connect"
+      },
+      "disconnect": {
+        "name": "Disconnect"
```

Comment:
There are references you can use `[%key:common::action::connect%]`

Reply:
There isn't one for disconnect right?

Reply:
There is, just add `dis`

Reply:
Where can I find a list of these references?

I can't find any reference to [%key:common::action::disconnect%]? Same for favorite

Reply:
Homeassistant/strings.json

There isn't one for favorites, hence I didn't include it in the code this comment is attached to

Reply:
Thanks!